src/julia-parser.scm

```diff
@@ -1577,8 +1577,12 @@
                       (else #t)))
          (rest  (if done
                     '()
-                    (parse-comma-separated s (lambda (s)
-                                               (parse-import s word))))))
+                    (let ((ex (parse-comma-separated s (lambda (s)
+                           (parse-import s word)))))
```

Comment:
Indentation needs to be fixed. (On the following lines as well.)

Reply:
So align `(parse-import s word)` to `(lambda`?